src/chrome/content/rules/Gocontigo.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Gocontigo.com">
```

Comment:
See also non-functional `sip.gocontigo.com`.

Reply:
Added, thank you.